setup.cfg

```diff
@@ -9,10 +9,6 @@ license = Apache License 2.0
 long_description = file: README.md
 long_description_content_type = text/markdown
 classifiers =
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
```

Comment:
Technically these also indicate the CLI version compatibility. But all the more testing to finally separate it :)